src/modules/iasql_functions/index.ts

```diff
@@ -64,3 +64,4 @@ export class IasqlFunctions extends ModuleBase {
  * ```
  */
 export const iasqlFunctions = new IasqlFunctions();
+export { getInstalledModules } from './iasql';
```

Comment:
I don't get this one. Why not follow the pattern in [other RPCs and just import `../iasql` directly](https://github.com/iasql/iasql/blob/main/src/modules/iasql_functions/rpcs/iasql_install.ts#L12)

Reply:
Yes it seemed a bit awkward to me as well. I'll move to that pattern.